### ISSUE TITLE
Make the submit buttons in the home page like the toggle buttons in the timeline page

### ISSUE THREAD

**Issue by Driexus** [Driexus/SimpleMoneyTracker#26]

**Comment by MohitKambli** [comment]
Hello,
Hope you're doing well.
Is this how the buttons in the home page should look like?

![image](https://github.com/Driexus/SimpleMoneyTracker/assets/31406633/bbcb57cf-6888-4c4c-b9f3-039cc66e114e)

I tried making the home page buttons and timeline page buttons the same, but I couldn't figure out the vertical spacing between them in the home page.
Let me know your thoughts on the same.

**Comment by Driexus** [comment]
Hello,
Thank you for your work!
The buttons look ok for now.

If you want to fix the spacing, maybe you can try some of the solutions from this thread:  https://stackoverflow.com/questions/53141752/set-the-space-between-elements-in-row-flutter

When you are done (even without the spacing), make a pull request so i can merge your changes.

**Comment by MohitKambli** [comment]
Hello,
I have made the possible changes that would make the submit buttons in the home_page.dart look similar to the toggle buttons in the timeline_page.dart and raised a pull request too as you might have noticed.
So, as of now, the home page looks something like this:

![image](https://github.com/Driexus/SimpleMoneyTracker/assets/31406633/36fc1836-2573-4226-a529-b14c0be6462e)

I was facing a problem while adding spacing to the submit buttons, but then I figured out that the Border widget would just do the work to differentiate all of them.
Even though the buttons' layout and the buttons look similar, their code is quite different.
I have just made changes in the **"home_page.dart"** file located at _"SimpleMoneyTracker\lib\ui\home"_.
Hope that these code changes meet the requirements.

Thank you,
Mohit Kambli
